README.md

```diff
@@ -1,4 +1,6 @@
 # linux-system-roles/network
+![CodeQL](https://github.com/linux-system-roles/network/workflows/CodeQL/badge.svg) ![Python Unit Tests](https://github.com/linux-system-roles/network/workflows/Python%20Unit%20Tests/badge.svg)![ShellCheck](https://github.com/linux-system-roles/network/workflows/ShellCheck/badge.svg)![Ansible Lint](https://github.com/linux-system-roles/network/workflows/Ansible%20Lint/badge.svg) ![Ansible Test](https://github.com/linux-system-roles/network/workflows/Ansible%20Test/badge.svg) ![Markdown Lint](https://github.com/linux-system-roles/network/workflows/Markdown%20Lint/badge.svg) ![Woke](https://github.com/linux-system-roles/network/workflows/woke/badge.svg) 
+---
```

Comment:
The svg link for Woke is invalid. 
![image](https://github.com/linux-system-roles/network/assets/36004580/8dcb520f-a180-4735-af7d-1d0ca73f3e6a)